docs/pages/server-access/architecture.mdx

```diff
@@ -0,0 +1,228 @@
+
+Once Teleport has been installed, Nodes connected, and initial users and permission created any remaining ports can be closed on both the *Bastion Host* and the protected, private, resources.
+
+### Divided topology
+
+Many architectures are now being designed to mitigate security events or disasters by splitting up resources and permissions into minimal groupings.
+
+Divided topologies involve deliberately restricting traffic, filtering IP addresses, regulating permissions, and grouping resources to mitigate loss or access in the advent of a disaster or security event.
+
+Teleport divides and groups access across resources through [Labels](../admin-guide.mdx#labeling-nodes-and-applications) and Node-based clustering. 
+
+- Admins can dynamically modify which Nodes join or are a part of which cluster.
+- Labels provide convenient groupings when querying, introspecting, or visualizing resources without modifying underlying clusters or access.
+
+## Cluster introspection
+
+Teleport simplifies managing and introspecting your cluster.
+
+### Unified Resource Catalog
+
+Resource admins will likely use the Teleport Unified Resource Catalog to easily manage resources by cluster and [Label](../admin-guide.mdx#labeling-nodes-and-applications).
+
+The following `tctl` command:
+
+```bash
+# Display the Unified Resource Catalog
+sudo tctl nodes ls
+``` 
+
+will display the Unified Resource Catalog with all queried resources in one view:
+   
+```bash
+Nodename         UUID                                 Address            Labels                               
+---------------- ------------------------------------ ------------------ ------------------------------------ 
+ip-172-31-40-99  bf840c02-68f4-4fac-a9e6-30141e82d242 127.0.0.1:3022     env=example,hostname=ip-172-31-40-99 
+ip-172-31-39-190 d32945f4-6e35-4318-b8f6-198d7c1fbd4c 172.31.39.190:3022                                      
+```
+
+Note the "Labels" column on the farthest side.
+
+### Labels
+
+Labels are permanent or temporary taxonomic handles for grouping resources. They can be used to group resources temporarily, across multiple clusters, or to define a grouping useful for querying without modifying the underlying access permissions or cluster membership:
+
+- `tsh ssh root@env=example` returns the list of all resources with the `env=example` label specified in their `teleport.yaml` (`'env': 'example'`).
+- Suppose an IP address changes, an admin can quickly find the current Node with that label since it remains unchanged.
+- Suppose you need to query the SSH Node of each cluster. Give each SSH Node the same label so a single query will allow you to introspect those specific resources across all of your clusters.
+
+This was designed in tandem with the Unified Resource Catalog to make it easier to manage mass-scale deployments (common to many industries like IoT and manufacturing).
```

Comment:
I would add a link that labels can be both dynamic and static. Dynamic tokens can be very powerful, in providing context and combined with RBAC